app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :users
```

Comment:
- [ ] the post belongs to User through the author, kindly make sure to implement this so the association setup correctly 

```ruby
belongs_to :author, class_name: 'User'
```

Reply:
well resolved! ✔️